bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/type/ChannelGroupType.java

```diff
@@ -45,11 +46,12 @@
      * @throws IllegalArgumentException if the UID is null, or the label is null or empty
      */
     public ChannelGroupType(ChannelGroupTypeUID uid, boolean advanced, String label, String description,
-            List<ChannelDefinition> channelDefinitions) throws IllegalArgumentException {
+            String category, List<ChannelDefinition> channelDefinitions) throws IllegalArgumentException {
```

Comment:
Just a quick review comment, I saw it was marked wip but it caught my eyes, the category field should imho also be final and it is missing from the constructor its docblock.